tools/s3s3mirror/pom.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  (c) Copyright 2013 Jonathan Cobb
+  This code is available under the Apache License, version 2: http://www.apache.org/licenses/LICENSE-2.0.html
+-->
```

Comment:
Do we need to update any documentation elsewhere to note our usage of this (its licensing)?